syntaxes/fsh.tmLanguage.json

```diff
@@ -29,7 +29,56 @@
       "patterns": [
         {
           "name": "keyword.control.fsh",
-          "match": "\\b(Alias|Characteristics|Context|Expression|Description|Mixins|Severity|Target|Title|Usage|XPath)(?=\\s*:)\\b"
+          "match": "\\b(Alias|Characteristics|Expression|Description|Mixins|Severity|Target|Title|Usage|XPath)(?=\\s*:)\\b"
+        },
+        {
+          "begin": "\\b(Context)\\s?(:)\\s+",
+          "beginCaptures": {
+            "1": {
+              "name": "keyword.reserved.fsh"
```

Comment:
I think the "name" for the "Context" portion of the regex should be the same as the other keywords, `keyword.control.fsh`. When I use the Dark+ theme, the Context keyword is a different color from the other keywords. Screenshot for reference:

<img width="317" alt="image" src="https://github.com/standardhealth/vscode-language-fsh/assets/30803904/1cdfc239-a429-40d9-9321-9f69aef4e458">

```suggestion
              "name": "keyword.control.fsh"
```